web/modules/custom/dpl_update/dpl_update.install

```diff
@@ -104,7 +104,8 @@ function dpl_update_install(): string {
   $messages[] = dpl_update_update_10022();
   $messages[] = dpl_update_update_10023();
   $messages[] = dpl_update_update_10024();
-
+  // Do not uninstall dblog module on install. It is not installed by default.
+  // $messages[] = dpl_update_update_10025()
```

Comment:
I'm not sure I understand this part. If we don't want to uninstall the dblog module on install, why are we adding the function for doing it in the first place?

Maybe I misunderstand this?

Reply:
Fair question. It seems like a pattern has emerged where every update hook in `dpl_update` is also added to `dpl_update_install()`.

Going through the changes I do not understand why it is needed but everyone (including myself) has kept up this pattern for a while now.

I wanted to show that in this situation I had thought about the situation and it is intentional not to add the update hook here - even if `dpl_update_update_10026()` is added at a later point in time.